app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java

```diff
@@ -351,8 +382,10 @@ private void createPdf(String mFilename) {
         mPath = mPath + mFilename + mActivity.getString(R.string.pdf_ext);
         try {
             PDFUtils fileUtil = new PDFUtils(mActivity);
-            fileUtil.createPdf(new TextToPDFOptions(mFilename, PageSizeUtils.mPageSize, mPasswordProtected,
-                    mPassword, mTextFileUri, mFontSize, mFontFamily, mFontColor), mFileExtension);
+            TextToPDFOptions options = new TextToPDFOptions(mFilename, PageSizeUtils.mPageSize, mPasswordProtected,
+                    mPassword, mTextFileUri, mFontSize, mFontFamily, mFontColor);
+            options.setmPageColor(mPageColor);
```

Comment:
add `mPageColor` in constructor.

Reply:
then I will have to add mPageColor in TextToPdfOptions class... you told me to move it to PdfOptions file.
So it's not possible to do it

Reply:
Check other parameters of a constructor some of them are also the member of `PDFOptions`